DiscordIan/Module/OpenWeather.cs

```diff
@@ -0,0 +1,317 @@
+            {
+                throw new Exception("Geocoding is not configured.");
+            }
+
+            var uri = new Uri(string.Format(_options.IanGeocodioEndpoint,
+                _options.IanGeocodioKey,
+                location));
+
+            var response = await _fetchJsonService.GetAsync<Model.Geocodio.Response>(uri);
+
+            if (response.IsSuccessful)
+            {
+                if (response?.Data?.Results != null)
+                {
+                    geocode = response?.Data?.Results[0]?.Location;
+                    address = response?.Data?.Results[0]?.AddressComponents;
+                }
+            }
+            else
+            {
+                throw new Exception(response.Message);
+            }
+
+            return (geocode, address);
+        }
+
+        private string FormatResults(WeatherCurrent.Current data, string location, WeatherForecast.Daily foreData = null)
+        {
+            var sb = new StringBuilder()
+                    .AppendLine(String.Format(">>> **{0}:** {1} {2}",
+                        location,
+                        TitleCase(data.Weather.Value),
+                        WeatherIcon(data.Weather.Icon)))
+
+                    .AppendLine(String.Format("**Temp:** {0}°{1} **Feels Like:** {2}°{3} **Humidity:** {4}{5}",
+                        data.Temperature.Value,
+                        data.Temperature.Unit.ToUpper()[0],
+                        data.Feels_like.Value,
+                        data.Feels_like.Unit.ToUpper()[0],
+                        data.Humidity.Value,
+                        data.Humidity.Unit))
+
+                    .Append(String.Format("**Wind:** {0} **Speed:** {1}{2} {3}",
+                        data.Wind.Speed.Name,
+                        data.Wind.Speed.Value,
+                        data.Wind.Speed.Unit,
+                        data.Wind.Direction.Code));
+
+            if (data.Wind.Gusts != "")
+            {
+                sb.Append(String.Format(" **Gusts:** {0}{1}",
+                        data.Wind.Gusts,
+                        data.Wind.Speed.Unit));
+            }
+
+            if (foreData != null)
+            {
+                sb.AppendLine()
+                    .AppendLine(String.Format("**High:** {0}°{1} **Low:** {2}°{3}",
+                        foreData.Temp.Max.ToString(),
+                        data.Temperature.Unit.ToUpper()[0],
+                        foreData.Temp.Min.ToString(),
+                        data.Temperature.Unit.ToUpper()[0]))
+                    .Append(String.Format("**Forecast:** {0} {1}",
+                        TitleCase(foreData.Weather[0].Description),
+                        WeatherIcon(foreData.Weather[0].Icon)));
+            }
+
+            return sb.ToString().Trim();
+        }
+
+        private string TitleCase(string str)
+        {
+            TextInfo textInfo = new CultureInfo("en-US", false).TextInfo;
```

Comment:
You can probably use [CultureInfo.CurrentCulture](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.currentculture?view=netcore-3.1).